packages/tooltip/usePopper.js

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
If this is only for storybook examples could we move this to a separate `utils` folder? This would be really helpful, but probably shouldn't be under a specific, versioned package.

Reply:
yeah was unsure if there is best practice around code just for storybook examples I'll move it elsewhere.

Reply:
would it make sense to create a `stories` folder with:
* `index.js` and
* `usePopper.js`?